weave-js/src/components/RunState.tsx

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
Note that this file is brought over unchanged from the core repo so that we can use it in Weave.